starter-code/pizza.js

```diff
@@ -1 +1,36 @@
 // Write your Pizza Builder JavaScript in this file.
+function main() {
+  $(".btn-pepperonni").on('click', function() {
+    $(this).toggleClass('active');
+    $(".pep").toggle();
+  });
+
+  $(".btn-mushrooms").on('click', function() {
+    $(this).toggleClass('active');
+    $(".mushroom").toggle();
+  });
+
+  $(".btn-green-peppers").on('click', function() {
+    $(this).toggleClass('active');
+    $(".green-pepper").toggle();
+  });
+
+
+  $(".btn-crust").on('click', function() {
+    $(this).toggleClass('active');
+    $('.crust-gluten-free').toggleClass("crust-gluten-free");
```

Comment:
For this option (also for white sauce) you are not going to toggle the same class that is query (in that case you could write the same code that before), instead:
```
$('.crust').toggleClass("crust-gluten-free");
```